app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java

```diff
@@ -176,7 +176,7 @@ public static Bitmap decodeFile(File f) {
      */
 
     public static boolean isApplicationInstalled(Context context, String packageName) {
-    private boolean isApplicationInstalled(Context context, String packageName) {
+    //private boolean isApplicationInstalled(Context context, String packageName) {
```

Comment:
Why changing the visibility to public ? is it used out side ?
if yes could you remove the commented code please.

Reply:
It was commented because it was crashing the app. Not sure how the duplicate got there in the first place.

Reply:
Yeah, I'm not sure. The duplicate is preventing building.